pages/public_cloud/ai_machine_learning/notebook_tuto_15_rock-paper-scissors/guide.en-gb.md

```diff
@@ -0,0 +1,114 @@
+---
+title: AI Notebooks - Tutorial - Train YOLOv8 to play to "rock paper scissors"
+excerpt:  How to train the YOLOv8 model on a custom dataset with AI Notebooks
+updated: 2023-09-01
+---
+
+## Objective
+
+
+The purpose of this tutorial is to show how it is possible to train YOLOv8 to play to the game "rock paper scissors"!
+YOLOv8 is an object detection algorithm. Although closely related to image classification, object detection performs image classification on a more precise scale. Object detection locates and categorizes features in images.
+
+It is based on the YOLOv8 open source [repository](https://github.com/ultralytics/ultralytics).
+
+## Requirements
+
+- Access to the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB)
+- An AI Notebooks project created inside a [Public Cloud project](https://www.ovhcloud.com/en-gb/public-cloud/) in your OVHcloud account
+- A user for AI Notebooks
+- A [Public Roboflow](https://public.roboflow.com/) account to access and download the [Rock Paper Scissors Dataset](https://universe.roboflow.com/roboflow-58fyf/rock-paper-scissors-sxsw)
+
+## Instructions
+
+This tutorial is based on the [Rock Paper Scissors SXSW Computer Vision Project Dataset](https://universe.roboflow.com/roboflow-58fyf/rock-paper-scissors-sxsw). It is available for free on [Roboflow](https://public.roboflow.com/).
+
+> [!primary]
+>
+> To learn more about the license of this dataset, please follow this [link](https://creativecommons.org/publicdomain/zero/1.0/).
+>
+
+### Create Object Storage containers
+
+If you want to create it from the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB), go to the Object Storage section and create a new object container by clicking `Object Storage` > `Create an object container`.
```

Comment:
Maybe add a short explanation about why would we need to create an object storage container

Reply:
It's explains just below 😉